src/dmd/link.d

```diff
@@ -853,10 +855,14 @@ public int runProgram()
     //printf("runProgram()\n");
     if (global.params.verbose)
     {
-        fprintf(global.stdmsg, "%s", global.params.exefile);
+        OutBuffer buf;
+        buf.writestring(global.params.exefile);
         for (size_t i = 0; i < global.params.runargs.dim; ++i)
```

Comment:
As I see this - FYI: there's as `asDArray` since a few weeks:

```d
foreach (param; global.params.runargs.asDArray)
    ...
```

Reply:
Sure, but that's outside the scope of this PR.